src/main/java/com/jlr/ttl/poc/loadmanager/models/Load_status.java

```diff
@@ -0,0 +1,63 @@
+package com.jlr.ttl.poc.loadmanager.models;
+
+
+import jakarta.persistence.*;
+
+import java.util.Date;
+
+@Entity
+@Table(name = "load_status")
+public class Load_status {
+
+    @Id
+    @Column(name = "id")
+    private String status_code;
+
+    @Column
+    private String description;
+
+    @Column(name = "end_date")
+    private Date date;
+
+    public Load_status() {
```

Comment:
Remove constructors and getters and setters. Use the following annotations instead:

AllArgsConstructor
NoArgsConstrutctor
Data
Build